_data/guides.yml

```diff
@@ -10,8 +10,10 @@ toc:
     path: /docs/whatisk8s/
   - title: Downloading or Building Kubernetes
     path: /docs/getting-started-guides/binary_release/
-  - title: Hello World Walkthrough
+  - title: Hello World Walkthrough on GKE
```

Comment:
The title gets a bit long, but maybe "Hello World on Google Container Engine" will fit